library/Mockery/Expectation.php

```diff
@@ -566,6 +566,9 @@ public function times($limit = null)
         if (is_null($limit)) {
             return $this;
         }
+        if (!is_int($limit)) {
+            throw new Exception('The passed Times limit should be an integer value');
```

Comment:
I think the exception thrown here is too generic. In my opinion it'd be much more expressive to throw an `InvalidArgumentException`.